docs/content/how-to/openstack/create-openstack-cluster.md

```diff
@@ -192,6 +193,129 @@ NAME      VERSION       AVAILABLE   PROGRESSING   SINCE   STATUS
 version   4.17.0        True        False         5m39s   Cluster version is 4.17.0
 ```
 
+## Enabling Ingress
+
+Note: In 4.17, enabling Ingress is a day 2 operation and we tested it with MetalLB. This might change in the future.
+
+* Create a floating IP in your OpenStack cloud that will be the public IP for Ingress.
```

Comment:
The Floating IP needs to be routed to the machine Network, right?
It might be worth to add that as a requirement.

Reply:
right but to me that goes without saying, since CAPO would connect the network to the external network via a router. Please suggest an addition if you feel like I missed something.

Reply:
```suggestion
* Create a floating IP using the external network configured on your cluster that will be the public IP for Ingress.
```